exercises/hangman/example.py

```diff
@@ -0,0 +1,60 @@
+import random
+
+
+def main():
+    while True:
```

Comment:
I don't like this infinite loop

Reply:
@m-a-ge This loop is only if player wan't to play again